src/pybind/mgr/dashboard/module.py

```diff
@@ -406,6 +411,40 @@ def set_mgr_created_self_signed_cert(self):
             return result
         return 0, 'Self-signed certificate created', ''
 
+    @CLICommand("dashboard get issue")
+    def get_issues_cli(self, issue_number: str):
+        if issue_number == 'None':
+            return -errno.EINVAL, '', 'Invalid arguments. Issue number is missing'
+        cephTrackerClient = CephTrackerClient()
```

Comment:
```suggestion
        tracker_client = CephTrackerClient()
```
We use `snake_case` for python vars.